doc/all.mk

```diff
@@ -45,10 +45,10 @@ CONF_FILES := $(filter-out %~,$(wildcard raddb/*conf raddb/mods-available/* radd
 BASE_ADOC_FILES := $(wildcard doc/*/*.adoc) $(wildcard doc/*/*/*.adoc)
 AUTO_ADOC_FILES := $(patsubst raddb/%,doc/raddb/%.adoc,$(CONF_FILES))
 AUTO_ADOC_FILES += $(patsubst raddb/%.md,doc/raddb/%.adoc,$(shell find raddb -name "*\.md" -print))
-AUTO_ADOC_FILES += $(patsubst doc/%.md,doc/%.adoc,$(wildcard doc/*/*/*.md))
+AUTO_ADOC_FILES += $(patsubst doc/%.md,doc/%.adoc,$(wildcard doc/*.md doc/*/*.md doc/*/*/*.md doc/*/*/*/*.md))
 ADOC_FILES	:= $(BASE_ADOC_FILES) $(AUTO_ADOC_FILES)
 PDF_FILES := $(patsubst doc/%.adoc,doc/%.pdf,$(ADOC_FILES))
-HTML_FILES := $(patsubst doc/%.adoc,doc/%.html,$(ADOC_FILES))
+HTML_FILES := $(patsubst doc/%.adoc,doc/%.html,$(ADOC_FILES)) $(subst %README.adoc,index.html,$(ADOC_FILES))
```

Comment:
@alandekok here, basically we are adding the `index.html` (symbolik-links to README.html) to be resolved during `install.doc` and `clean.doc`